src/Arcus.Observability.Telemetry.Core/Extensions/ILoggerExtensions.cs

```diff
@@ -31,12 +43,19 @@ public static void LogRequest(this ILogger logger, HttpRequest request, HttpResp
             var resourcePath = request.Path;
             var host = $"{request.Scheme}://{request.Host}";
 
-            logger.LogInformation(
-                $"{MessagePrefixes.RequestViaHttp} {{{ContextProperties.RequestTracking.RequestMethod}}} {{{ContextProperties.RequestTracking.RequestHost}}}/{{{ContextProperties.RequestTracking.RequestUri}}} completed with {{{ContextProperties.RequestTracking.ResponseStatusCode}}} in {{{ContextProperties.RequestTracking.RequestDuration}}} at {{{ContextProperties.RequestTracking.RequestTime}}} (Context: {{@{ContextProperties.EventTracking.EventContext}}})",
-                request.Method,
-                host, resourcePath, statusCode, duration, DateTimeOffset.UtcNow, context);
+            logger.LogInformation(RequestFormat, request.Method, host, resourcePath, statusCode, duration, DateTimeOffset.UtcNow, context);
         }
 
+        private const string HttpDependencyFormat =
```

Comment:
Can we move all of these to the top please, easier to see all constants that are available